src/transformers/models/auto/feature_extraction_auto.py

```diff
@@ -75,8 +75,10 @@ def feature_extractor_class_from_name(class_name: str):
             module_name = model_type_to_module_name(module_name)
 
             module = importlib.import_module(f".{module_name}", "transformers.models")
-            return getattr(module, class_name)
-            break
+            try:
+                return getattr(module, class_name)
+            except AttributeError:
+                continue
```

Comment:
With the sort, sometimes the module in which the class is does not appear first, so we must continue iterating.